### PR TITLE
fix: make hover card tappable

### PR DESCRIPTION
**Pull request by remonke**
Let's just remove this.

In Radix Primitives for React this doesn't even do anything as the `event` there is some weird React abstraction.

### CODE REVIEW

**Comment by remonke** [comment]
#802

**Comment by ChrisGV04** [comment]
Looks great!

I didn't see this PR and opened another one almost identical, but I already closed it. Sorry.

I also suggest replacing the end of docs with the real keyboard interactions: `docs/content/components/hover-card.md`

```md
## Accessibility

The hover card is intended for sighted users only, the content will be inaccessible to keyboard users.

### Keyboard Interactions

<KeyboardTable :data="[
    {
      keys: ['Tab'],
      description: 'Opens/closes the hover card.',
    },
    {
      keys: ['Enter'],
      description: 'Opens the hover card link',
    }]" />
```

This comes from Radix UI.

File: packages/radix-vue/src/HoverCard/utils.ts
Comment (zernonia) [inline packages/radix-vue/src/HoverCard/utils.ts:1]: let's revert this changes. Wanna keep the similar callback reference, and don't think this is causing any issue

**Review by zernonia — COMMENTED** [review]
Thanks for the PR @remonke ! Happy with removing `@touchstart.prevent`, however I'll like to keep the utility function the same.

And let's add the docs regarding the Accessibility as suggested by @ChrisGV04  😁

**Review by zernonia — APPROVED** [review]
Thanks for the PR @remonke . Reverted `@pointerleave` as it overwrote the `GraceArea` feature of HoverCard
